### PR TITLE
Added instructions for adding New Model

### PR DESCRIPTION
**Pull request by gautamjajoo**
Added instructions for adding a New Model.
Thanks!

### CODE REVIEW

File: tutorials/AddingNewModel.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<h1>Adding New Model - Caffe</h1>
+
+1. For Setup Instructions. - https://github.com/Cloud-CV/Fabrik/blob/master/README.md
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: Modify to: For setup instructions, look at the [README](https://github.com/Cloud-CV/Fabrik/blob/master/README.md)

File: tutorials/AddingNewModel.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+2. Open the <b>Example folder</b>.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: Add a relative link to the `example` folder

File: tutorials/AddingNewModel.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+3. Open the <b>Caffe folder</b> and add the new model in the form of .prototxt file.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: Add a relative link to the caffe folder

File: tutorials/AddingNewModel.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+4. Then add your entry to the frontend by adding the following line to <b>topbar.js</b>, in this the id should be the name of your prototxt without the extension.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: – *front-end
– add a link to topbar.js
– use code highlighting on `id`, and reference the example provided below

File: tutorials/AddingNewModel.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+     - The location of topbar.js is <b>ide/static/js/topbar.js</b>
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: delete

File: tutorials/AddingNewModel.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+1. For Setup Instructions. - https://github.com/Cloud-CV/Fabrik/blob/master/README.md
+2. Open the <b>Example folder</b>.
+3. Open the <b>Caffe folder</b> and add the new model in the form of .prototxt file.
+4. Then add your entry to the frontend by adding the following line to <b>topbar.js</b>, in this the id should be the name of your prototxt without the extension.
+     - The location of topbar.js is <b>ide/static/js/topbar.js</b> 
+```
+<li><ModelElement importNet={this.props.importNet} framework="caffe" id="sample">sample</ModelElement></li>
+
+```
+5. After making these changes, test if loading the model and exporting it to both or at least one framework is working fine.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: .. and document it accordingly in your pull request

File: tutorials/AddingNewModel.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+6. Create a pull request for the same and get reviewed by the mentors.
+Cheers!
+
+<h1>Adding New Model - Keras </h1>
+
+1. For Setup Instructions. - https://github.com/Cloud-CV/Fabrik/blob/master/README.md
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: Use markdown on links as above, not full paths

File: tutorials/AddingNewModel.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+2. Open the <b>Example folder.</b>
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: same as above

File: tutorials/AddingNewModel.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+3. Open the <b>Keras folder</b> and add the new model in the form of .json file.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: same as above

File: tutorials/AddingNewModel.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+4. Then add your entry to the frontend by adding the following line to <b>topbar.js</b>, in this the id should be the name of your json without the extension.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: same as above

File: tutorials/AddingNewModel.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+    - The location of topbar.js is <b>ide/static/js/topbar.js</b>
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: delete

File: tutorials/AddingNewModel.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+```
+<li><ModelElement importNet={this.props.importNet} framework="keras" id="sample">sample</ModelElement></li> 
+```
+5. After making these changes, test if loading the model and exporting it to both or at least one framework is working fine.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: same as above

File: tutorials/AddingNewModel.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
+<h1>Adding New Model - Caffe</h1>
+
+1. For Setup instructions, look at the [README](https://github.com/Cloud-CV/Fabrik/blob/master/README.md).
+2. Open the <b>[Example folder](https://github.com/Cloud-CV/Fabrik/tree/master/example)</b>.
+3. Open the <b>[Caffe folder](https://github.com/Cloud-CV/Fabrik/tree/master/example/caffe)</b> and add the new model in the form of .prototxt file.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: What I earlier meant was to combine 2 and 3, and just say something like add the new model as a prototxt file to the `example/caffe` folder. Can you make that small change?

File: tutorials/AddingNewModel.md [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+```
+5. After making these changes, test if loading the model and exporting it to both or at least one framework is working fine and document it accordingly in your pull request.
+6. Create a pull request for the same and get reviewed by the mentors.
+Cheers!
+
+<h1>Adding New Model - Keras </h1>
+
+1. For Setup instructions, look at the [README](https://github.com/Cloud-CV/Fabrik/blob/master/README.md).
+2. Open the <b>[Example folder](https://github.com/Cloud-CV/Fabrik/tree/master/example)</b>.
+3. Open the <b>[Keras folder](https://github.com/Cloud-CV/Fabrik/tree/master/example/keras)</b> and add the new model in the form of .json file.
```
Comment (virajprabhu) [inline tutorials/AddingNewModel.md]: Same comment as above

**Comment by utsavgarg** [comment]
LGTM :)

**Pull request merged**